docs/administration/configuration/config-file-reference.md

```diff
@@ -624,6 +624,23 @@ because the File Upload and Job Run are performed as separate steps.)
 # default is 10 minutes
 rundeck.fileUploadService.tempfile.expiration=600000
 ```
+### File Size Max
+Specifies the size of the largest request the server can accept
+
+For 1GB ~ 2GB files, is recommended to set:
+```properties
+grails.controllers.upload.maxFileSize=4096000000
+grails.controllers.upload.maxRequestSize=4096000000
+rundeck.fileUploadService.tempfile.maxsize=4096000000
+rundeck.fileUploadService.tempfile.expiration=600000
```

Comment:
Is this in seconds, milliseconds?  Let's add a line somewhere in this update that clarifies what the values represent for each setting.

Reply:
**Hi**, the added module is removed and an existing one is used that deals with the same thing. Descriptions are added that represent each value in the configurations.

### Old configuration:
<img width="786" alt="image" src="https://user-images.githubusercontent.com/91557307/172759467-52ad6c71-6d3f-45d8-ad96-f8543fa3a46c.png">

### New configuration:
<img width="802" alt="image" src="https://user-images.githubusercontent.com/91557307/172759667-8e563412-0b24-4186-8959-4b4072f7d353.png">